spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/NativeEnvironmentRepository.java

```diff
@@ -264,6 +264,8 @@ protected Environment clean(Environment env,
 					locations = new String[] { matcher.group(2) };
 				}
 			}
+			name = name.replace("\\", "/"); // change windows path '\' into '/'
```

Comment:
Doesn't this effect all OSes?

Reply:
Only Windows uses `\` as path separator, so `name = name.replace("\\", "/")` only effect Windows.